test_runner/src/main/kotlin/ftl/run/platform/RunAndroidTests.kt

```diff
@@ -29,36 +31,37 @@ internal suspend fun runAndroidTests(args: AndroidArgs): TestResult = coroutineS
 
     // GcAndroidMatrix => GcAndroidTestMatrix
     // GcAndroidTestMatrix.execute() 3x retry => matrix id (string)
-    val androidDeviceList = GcAndroidDevice.build(args.devices)
-
     val testMatrices = mutableListOf<Deferred<TestMatrix>>()
     val allTestShardChunks = mutableListOf<List<String>>()
     val ignoredTestsShardChunks = mutableListOf<List<String>>()
 
     val history = GcToolResults.createToolResultsHistory(args)
     val otherGcsFiles = args.uploadOtherFiles(runGcsPath)
     val additionalApks = args.uploadAdditionalApks(runGcsPath)
+    val argsList = if (args.shouldSplitRuns()) args.splitConfigurationByDeviceType() else listOf(args)
```

Comment:
`argsList` is a little bit generic in this context maybe `deviceConfigurations` or `deviceArgs` will fits better?

Reply:
`deviceArgs` fits great, thanks!